### PR TITLE
style fix width of input picture

### PR DESCRIPTION
**Pull request by mrhut10**
fixed width of input picture within the sidebar.

method:
  unwraped it from a feildset, which fixed it.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -30,10 +30,7 @@
   background:white;
   grid-area: settings;
   padding:10px;
-  /*
-  display: none;
-  visibility: hidden;
-  */
+  overflow:hidden;
```
Comment (khutten) [inline css/main.css]: Comment: There is no space between the colon and 'hidden', maybe we want to set up a linter?

Comment (mrhut10) [reply css/main.css]: i think that has been fixed now? can you see it @khutten

**Review by mrhut10 — COMMENTED** [review]
set leading space on css properties

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by khutten** [comment]
@mrhut10 I can see the leading spaces being set now, though not sure if that's a good thing. I more ment maybe it's time we set up a linter. Not can you change these all by hand please. All of the spaces being updated in this PR made it more difficult to review since there was a lot more red and green around, ya know.

**Pull request merged**